src/OidcClient/TokenRefreshedEventArgs.cs

```diff
@@ -17,10 +17,11 @@ public class TokenRefreshedEventArgs : EventArgs
         /// <param name="accessToken">The access token.</param>
         /// <param name="refreshToken">The refresh token.</param>
         /// <param name="expiresIn">The expires in.</param>
-        public TokenRefreshedEventArgs(string accessToken, string refreshToken, int expiresIn)
+        public TokenRefreshedEventArgs(string accessToken, string refreshToken, string identityToken, int expiresIn)
```

Comment:
The problem is that this is introducing a breaking change because you changed the signature.

I would prefer to move identityToken to the end of the param list and make it optional.

Reply:
Understood. New changes pushed